src/spikeinterface/core/generate.py

```diff
@@ -831,6 +831,52 @@ def clean_refractory_period(times, refractory_period):
     return times
 
 
+def _add_insertions(spike_train, insertions=None, insertions_replace=False, seed=None):
```

Comment:
```suggestion
def add_spikes_to_spiketrain(spike_indices, spike_labels, segment_indices=None, added_spike_indices=None, added_spike_labels=None, added_segment_indices=None, replace=False, seed=None):
```

sorting.to_spike_vector()